tools/nni_cmd/nnictl_utils.py

```diff
@@ -929,7 +942,26 @@ def load_experiment(args):
             else:
                 shutil.copy(src_path, target_path)
 
-    # Step5. Create experiment metadata
+    # Step5. Copy searchSpace file
+    archive_search_space_dir = os.path.join(temp_root_dir, 'searchSpace')
+    target_path = os.path.expanduser(args.searchSpacePath)
+    if not os.path.isabs(target_path):
+        target_path = os.path.join(os.getcwd(), target_path)
+        print_normal('Expand search space path to %s' % target_path)
+    nnictl_exp_config['searchSpacePath'] = target_path
+    # if the path alerady has a search space file, use the original one, otherwise use archived one
+    if not os.path.isfile(target_path):
+        if len(os.listdir(archive_search_space_dir)) == 0:
+            print_error('Archive file does not contain search space file!')
+            exit(1)
+        else:
+            for file in os.listdir(archive_search_space_dir):
+                source_path = os.path.join(archive_search_space_dir, file)
+                os.makedirs(os.path.dirname(target_path), exist_ok=True)
+                shutil.copyfile(source_path, target_path)
+                break
```

Comment:
Throw error when no file is found?

Reply:
there is already a judgement in lint 955, don't need to throw error here anymore.